src/main/java/delete_event_use_case/DeleteEventPresenter.java

```diff
@@ -0,0 +1,14 @@
+package delete_event_use_case;
+
+/**
+ * @author : Name
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
+public class DeleteEventPresenter implements DeleteEventOutputBoundary {
+    @Override
+    public DeleteEventOutputData prepareSuccessView(DeleteEventOutputData response) {
+        //TODO: View Model Update?
+        return response;
+    }
```

Comment:
Let's update the success here so we know the use case was successfully done. This will follow the SOLID principles, precisely the Single Responsibility Principle where all it will do is update the attribute so that later the GUI can check the attribute to update UI as desired. You can ask Anna more about the UI details since she is working on that Use Case.